build-system/config.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
     // Files in this testdata dir are machine-generated and are not part
     // of the AMP runtime, so shouldn't be checked.
     '!extensions/amp-a4a/*/test/testdata/*.js',
-    '!examples/*.js',
+    '!examples/**/*',
```

Comment:
@rsimha FYI

Reply:
Gotcha. Not sure why we were running presubmit checks on `examples` in the first place.